### PR TITLE
Added Color supporting to adminchat

### PR DESCRIPTION
**Pull request by ghost**
Added Color supporting to adminchat.
Just some small fun to admins.

### CODE REVIEW

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_adminchat.java [hunk 1]
```diff
@@ -15,7 +15,6 @@
         aliases = "o,ac")
 public class Command_adminchat extends FreedomCommand
 {
-
```
Comment (OxLemonxO) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_adminchat.java]: What's the reason of the removal of this space?

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_adminchat.java [hunk 2]
```diff
@@ -33,9 +32,13 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
...
         {
-            plugin.cm.adminChat(sender, StringUtils.join(args, " "));
+            if(StringUtils.join(args, " ").contains("&k") || StringUtils.join(args, " ").contains("&0") || StringUtils.join(args, " ").contains("&m"))
+            {
+                msg("You can't use Forbidden Colors!");
+                return true; 
+            }
+            plugin.cm.adminChat(sender, FUtil.colorize(StringUtils.join(args, " ")));
         }
-
```
Comment (OxLemonxO) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_adminchat.java]: Also this

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_adminchat.java [hunk 3]
```diff
@@ -33,9 +32,13 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
         }
         else
         {
-            plugin.cm.adminChat(sender, StringUtils.join(args, " "));
+            if(StringUtils.join(args, " ").contains("&k") || StringUtils.join(args, " ").contains("&0") || StringUtils.join(args, " ").contains("&m"))
```
Comment (OxLemonxO) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_adminchat.java]: Why not create a variable for the args, and then check if it contains that?

**Comment by marcocorriero** [comment]
@OxLemonxO im not a good dev, i did my best to make them real, if you want to help implementing the code, it will be helpful

**Comment by Wild1145** [comment]
Thank you for contributing to the project, could you please amend your opening message to include the issue that this PR resolves. Thank You.

**Comment by ghost** [comment]
Marco, please stick with making macros and not Java...

**Comment by ghost** [comment]
You made that all too complicated... FUtil.colorize, thats all I have to say...

**Comment by ghost** [comment]
@Commodore64x java is not simple

**Comment by ghost** [comment]
@marcocorriero it is simple for me. I think much like the language.

**Comment by ghost** [comment]
No, is not, futil colorize is coded by prozza.

Inviato da iPad

Il giorno 10 nov 2016, alle ore 07:35, Commodore64x <notifications@github.com<mailto:notifications@github.com>> ha scritto:

@marcocorrierohttps://github.com/marcocorriero it is simple for me. I think much like the language.

##

**Comment by ghost** [comment]
So? What is the point of having any methods in the TFM at all if they don't get used...

**Comment by ghost** [comment]
@Commodore64x i used futil.colorize event coded by prozza...

**Comment by Wild1145** [comment]
This PR is on hold pending an associated issue.

**Pull request closed without merging**